packages/now-build-utils/src/types.ts

```diff
@@ -99,3 +99,26 @@ export interface PrepareCacheOptions {
    */
   config: Config;
 }
+
+export interface ShouldServeParams {
+  /**
+   * All source files of the project
+   */
+  files: Files;
+  /**
+   * Name of entrypoint file for this particular build job. Value
+   * `files[entrypoint]` is guaranteed to exist and be a valid File reference.
+   * `entrypoint` is always a discrete file and never a glob, since globs are
+   * expanded into separate builds at deployment time.
+   */
+  entrypoint: string;
+  /**
+   * A path string for a request.
+   */
+  requestPath: string;
+  /**
+   * An arbitrary object passed by the user in the build definition defined
+   * in `now.json`.
+   */
+  config?: object;
```

Comment:
```suggestion
  config: Config;
```

Should this be the same config from the `BuildOptions`?

Reply:
Yes it is 👍
but it should be optional

Reply:
@amio Why should it be optional?

We intentionally made the config object always exists before calling the builder.

https://github.com/zeit/lambdas/pull/368

Reply:
In now-dev's case it's omitted 🤔
https://github.com/zeit/now-cli/blob/0f74a0695c6de3605ebca616ec373c9cc7c7cca1/src/commands/dev/lib/dev-server.ts#L680

Reply:
@amio Ok then let's remove line 123 because it looks like there is never a config passed in.

Reply:
I guess so, but not sure if @TooTallNate had a plan for it

Reply:
It's listed here, in `static-builder`'s `ShouldServeParams`: https://github.com/zeit/now-cli/blob/0f74a0695c6de3605ebca616ec373c9cc7c7cca1/src/commands/dev/lib/types.ts#L115

Reply:
@amio I just deleted that property from the `now-cli` and everything built fine. I would leave it off because we can always add it later in a minor update but we can never remove it without a breaking change.